### PR TITLE
Add Unit Tests to TTS IsReady false

### PR DESCRIPTION
**Pull request by OHerasym**
### Pull request changes:
- Add unit tests for CRQ TTS IsReady false

Related: [APPLINK-25122](https://adc.luxoft.com/jira/browse/APPLINK-25122)

### CODE REVIEW

**Comment by OHerasym** [comment]
Please review: @AGritsevich , @vlantonov , @VProdanov , @anosach-luxoft , @orhan-mehmedov

File: src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc [hunk 1]
```diff
@@ -61,11 +62,59 @@ namespace am = ::application_manager;
...
+    utils::SharedPtr<AlertManeuverRequest> command =
+        CreateCommand<AlertManeuverRequest>(response);
+
+    MockAppPtr mock_app(CreateMockApp());
+    ON_CALL(app_mngr_, application(_)).WillByDefault(Return(mock_app));
+
+    MockMessageHelper* mock_message_helper =
+        MockMessageHelper::message_helper_mock();
+    EXPECT_CALL(*mock_message_helper, HMIToMobileResult(_))
+        .WillOnce(Return(mobile_apis::Result::UNSUPPORTED_RESOURCE));
```
Comment (VVeremjova) [inline src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc]: @OHerasym Why don't use `mobile_response` here?

Comment (OHerasym) [inline src/components/application_manager/test/commands/mobile/alert_maneuver_request_test.cc]: @VVeremjova , mobile_apis::Result::UNSUPPORTED_RESOURCE is used in all cases

File: src/components/application_manager/test/commands/mobile/set_global_properties_request_test.cc [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+using ::testing::_;
+using ::utils::SharedPtr;
+using ::testing::Return;
+using ::testing::ReturnRef;
+using am::commands::SetGlobalPropertiesRequest;
+using ::test::components::application_manager_test::MockApplication;
+
+class SetGlobalPropertiesRequestTest
+    : public CommandRequestTest<CommandsTestMocks::kIsNice> {
+ public:
```
Comment (OHerasym) [inline src/components/application_manager/test/commands/mobile/set_global_properties_request_test.cc]: @VVeremjova , Test is using default ctor from Test Class

File: src/components/application_manager/test/commands/mobile/set_global_properties_request_test.cc [hunk 3]
```diff
@@ -0,0 +1,151 @@
...
+};
+
+TEST_F(SetGlobalPropertiesRequestTest, Test1) {
```
Comment (OHerasym) [inline src/components/application_manager/test/commands/mobile/set_global_properties_request_test.cc]: @VVeremjova , Done, db49726

File: src/components/application_manager/test/commands/mobile/speak_request_test.cc [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+
+  EXPECT_EQ(
+      (*response_to_mobile)[strings::msg_params][strings::success].asBool(),
+      true);
+  EXPECT_EQ(
+      (*response_to_mobile)[strings::msg_params][strings::result_code].asInt(),
+      static_cast<int32_t>(mobile_apis::Result::SUCCESS));
+}
+
+TEST_F(SpeakRequestTest, OnEvent_UNSUPPORTED_RESOURCE_Case1) {
```
Comment (OHerasym) [inline src/components/application_manager/test/commands/mobile/speak_request_test.cc]: @VVeremjova , Done, db49726

**Pull request merged**